lib/rollbar_helper.rb

```diff
@@ -3,55 +3,65 @@
 
 module RollbarHelper
   LEVELS = [
-    :critical,
     :debug,
-    :error,
     :info,
     :warning,
-  ]
+    :error,
+    :critical,
+  ].freeze
 
   class << self
-    def critical(obj, fingerprint: nil, **data)
-      log(:critical, obj, :callee => caller, fingerprint: fingerprint, **data)
-    end
-
-    def debug(obj, fingerprint: nil, **data)
-      log(:debug, obj, :callee => caller, fingerprint: fingerprint, **data)
-    end
+    LEVELS.each do |level|
+      define_method(level) do |*args|
+        message, exception, extra = extract_arguments(args)
+        extra = {callee: caller}.merge(extra)
+        data, exception, callee, fingerprint = split_extra_arg(extra, exception)
```

Comment:
is L17 needed to allow people to run `RollbarHelper.whatever(..., :callee => something)`?

Reply:
Yes, because of the line above: `caller` will be used as the default `callee` value, but if `callee` is provided as part of a Hash, it will override that value.